src/components/i18nStrings.vue

```diff
@@ -135,11 +137,26 @@ export default {
 					this.$store.commit('cleanStateFor', this.path)
 				}
 			},
-      		deep: true
-		},
+			  deep: true,
```

Comment:
You have mixed indentation with spaces and tabs. Please change indentation to tabs here and other places in the commit. I'd recommend installing the ESLint extension for your editor, which makes these issues easier to spot.

Reply:
@tahme  Added extension and modified the impacted code again. Please have a look again :)